src/planscape/planscape/urls.py

```diff
@@ -29,4 +30,8 @@
     path('planscape-backend/dj-rest-auth/', include('dj_rest_auth.urls')),
     path('planscape-backend/dj-rest-auth/registration/',
          include('dj_rest_auth.registration.urls')),
+    # TODO: We don't actually need this since we'll be using
+    # a different endpoint.
+    path('planscape-backend/dj-rest-auth/password/reset/confirm/<uidb64>/<token>/',
```

Comment:
Is this uint64?

Reply:
I'm confused about the TODO that is here. Why is this being checked in if we are not using this endpoint?

Reply:
This was a temporary solution that doesn't apply anymore. I've removed the change.